src/util.hpp

```diff
@@ -115,10 +105,9 @@ std::pair<double, double> convert_vt_to_ll(std::uint32_t extent,
   The first point is considered the "origin" and its latitude is used to initialize
   the ruler. The second is considered the "feature" and is the distance to.
 */
-namespace cr = mapbox::cheap_ruler;
-double distance_in_meters(cr::point origin_lnglat, cr::point feature_lnglat) {
+double distance_in_meters(mapbox::geometry::point<double> origin_lnglat, mapbox::geometry::point<double> feature_lnglat) {
     // set up cheap ruler with query latitude
-    cr::CheapRuler ruler(origin_lnglat.y, cr::CheapRuler::Meters);
+    mapbox::cheap_ruler::CheapRuler ruler(origin_lnglat.y, mapbox::cheap_ruler::CheapRuler::Meters);
```

Comment:
If you wanted to use `cr::` just within the method here you could add `using cr = mapbox::cheap_ruler`  within the function and it would have a scope of only the function.